dll/win32/shell32/utils.cpp

```diff
@@ -204,6 +204,59 @@ HRESULT SHILAppend(_Inout_ LPITEMIDLIST pidl, _Inout_ LPITEMIDLIST *ppidl)
     return hr;
 }
 
+/*************************************************************************
+ *  InternalExtractIconListW [SHELL32.238]
+ */
+EXTERN_C
+HGLOBAL WINAPI
+InternalExtractIconListW(
+    _In_ HINSTANCE hInst,
+    _In_ PCWSTR pszExeFile,
+    _In_opt_ PUINT pUnused)
+{
+    UNREFERENCED_PARAMETER(pUnused);
+
+    SIZE_T cIcons = (SIZE_T)ExtractIconW(hInst, pszExeFile, 0xFFFFFFFF);
```

Comment:
```suggestion
    SIZE_T cIcons = (SIZE_T)ExtractIconW(hInst, pszExeFile, (UINT)-1);
```
or `MAXUINT`

I wonder whether you could limit the variable to UINT (always 32-bits) since the number of icons won't be larger than 2^32-1 , and, the `PrivateExtractIconsW` call below takes a UINT.

`cIcons` -> `nIcons` maybe (matches the name of the function parameter in msdn; 'n' usually means a quantity)...

Reply:
Fixed.